source/_integrations/mullvad.markdown

```diff
@@ -0,0 +1,26 @@
+---
+title: Mullvad VPN
+description: Instructions on how to integrate Mullvad VPN api data into Home Assistant.
+ha_category:
+  - Binary Sensor
+ha_iot_class: Cloud Polling
+ha_release: 0.114
+ha_domain: mullvad_vpn
+ha_codeowners:
+  - '@meichthys'
+---
+
+The Mullvad VPN integration pulls [Mullvad](https://mullvad.net/) VPN status information into Home Assistant.
+
+<p class='img'>
+<img src='/images/screenshots/mullvad_vpn_sample_sensor_connected.png' />
```

Comment:
There are 2 images attached to this PR, however, only 1 is referenced on this page.

Is there an image that needs to be removed from the PR?

Reply:
I think they were both referenced initially but for first-time PRs we are only allowed to have a single sensor so the second image was removed since only the binary-sensor is included in the initial PR. I'll remove it form this pr.